addons/im_livechat/static/src/embed/common/livechat_service.js

```diff
@@ -229,17 +229,7 @@ export class LivechatService {
     }
 
     async initializePersistedSession() {
-        if (this.guestToken) {
-            await this.busService.updateContext({
-                ...this.busService.context,
-                guest_token: this.guestToken,
-            });
-        }
-        if (this.busService.isActive) {
-            this.busService.forceUpdateChannels();
-        } else {
-            await this.busService.start();
-        }
+        await this.busService.addChannel(`mail.guest_${this.guestToken}`);
```

Comment:
Is this awaiting long enough for start to be done before resolving?
As to keep the same behavior as before, not calling init messaging before the bus is ready.

Reply:
Both `start` and `addChannel` are awaiting `connectionInitializedDeferred` so timing is unchanged.